index.js

```diff
@@ -187,7 +187,15 @@ async function generateWorkflow (file, options) {
       }
 
       if (swagger.tags && swagger.paths[path][method].tags) {
-        swagger.paths[path][method].tags.forEach(tag => workflow.tests[tag].steps.push(step))
+        swagger.paths[path][method].tags.forEach(tag => {
+          if(Object.keys(workflow.tests).includes(tag)){
+            workflow.tests[tag].steps.push(step)
+          }else{
+            workflow.tests[tag] = {
+              name: undefined,
```

Comment:
because I think name will have an description, is `undefined` correct ?
https://github.com/stepci/plugin-openapi/blob/30b6e56b7809ff12a253c1fe14ccde31993cc433/index.js#L60-L67